### PR TITLE
Fix for doubled dial code when separateDialCode is enabled

### PR DESCRIPTION
**Pull request by mirceaciu**
A quick fix for the doubled dial code of issue [#823](https://github.com/jackocnr/intl-tel-input/issues/823)

I've extended `_getFullNumber` to check for when a phone number starts with the dial code.

As a result this input:
image
 will output `+4071234567`

### CODE REVIEW

**Comment by iMojtaba** [comment]
Please merge this branch ...

**Comment by mirceaciu** [comment]
I've updated the PR as per recommendations.

side note: looking at the _startsWith method
```
  _startsWith(a, b) {
    return (a.substr(0, b.length).toLowerCase() === b);
  }
```
shouldn't `b` also be converted to lowercase so that apples are compared to apples?
  
 `return (a.substr(0, b.length).toLowerCase() === b.toLowerCase());`

Also [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) recommends using `substring` instead of `substr`. What's your take on this?

File: src/js/intlTelInput.js [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ class Iti {
 
     // normalized means ensure starts with a 1, so we can match against the full dial code
     const normalizedVal = (numericVal.charAt(0) === '1') ? numericVal : `1${numericVal}`;
-    if (this.options.separateDialCode) {
+    if (this.options.separateDialCode && val.charAt(0) !== '+') {
```
Comment (jackocnr) [inline src/js/intlTelInput.js]: 👌 perfect, thank you

**Comment by mirceaciu** [comment]
I'll take a better look at the code and look for `substr` and will make a PR on that(and maybe consider the _startsWith change then, if needed)

**Comment by jackocnr** [comment]
Brilliant thank you. I will sort out these build conflicts and get this merged over the weekend.

**Comment by jackocnr** [comment]
Merged. Thanks again.

**Pull request closed without merging**